### PR TITLE
Add template documentation

### PR DESCRIPTION
**Pull request by jmontoyaa**

### CODE REVIEW

File: Resources/doc/reference/installation.rst [hunk 1]
```diff
@@ -151,4 +151,27 @@ Enable the Timeline Block
 
 And now, you're good to go !
 
+Edit the Timeline Block
+-------------------------  
+
+.. configuration-block::
```
Comment (OskarStark) [inline Resources/doc/reference/installation.rst]: why the configuration block here? this should be done before the yaml code block

File: Resources/doc/reference/installation.rst [hunk 2]
```diff
@@ -151,4 +151,27 @@ Enable the Timeline Block
...
+
+Create a new template file here, based in the default timeline.html.twig
```
Comment (OskarStark) [inline Resources/doc/reference/installation.rst]: filename in ticks please

File: Resources/doc/reference/installation.rst [hunk 3]
```diff
@@ -151,4 +151,27 @@ Enable the Timeline Block
...
+
+  .. code-block:: yaml
+
+        # app/config/config.yml
+
+        sonata_admin:
+            dashboard:
+                blocks:
+                    # ...
+                    - { position: center, type: sonata.timeline.block.timeline, settings: { template: 'ApplicationTimelineBundle::Block:timeline.html.twig', context: SONATA_ADMIN, max_per_page: 25}}
```
Comment (OskarStark) [inline Resources/doc/reference/installation.rst]: space after `25` needed

File: Resources/doc/reference/installation.rst (hunk 2 above)
Comment (OskarStark) [inline Resources/doc/reference/installation.rst]: based ON, not IN

File: Resources/doc/reference/installation.rst [hunk 4]
```diff
@@ -151,4 +151,27 @@ Enable the Timeline Block
 
 And now, you're good to go !
 
+Edit the Timeline Block
+-------------------------
```
Comment (OskarStark) [inline Resources/doc/reference/installation.rst]: underline is too long

**Comment by OskarStark** [comment]
@jmontoyaa thank you for your contribution :+1:

**Comment by OskarStark** [comment]
@jmontoyaa thank you, can you please rebase the branch and fix the travis build?

**Comment by core23** [comment]
Please squash your commits @jmontoyaa

**Comment by OskarStark** [comment]
👍

**Pull request merged**